Sources/AppcuesKit/Presentation/Traits/Appcues/AppcuesEmbeddedTrait.swift

```diff
@@ -44,15 +44,17 @@ internal class AppcuesEmbeddedTrait: AppcuesStepDecoratingTrait, AppcuesContaine
         stepController.padding = NSDirectionalEdgeInsets(paddingFrom: style)
     }
 
+    @MainActor
```

Comment:
Correct, they're not required since it's implicit from the protocol definition. I like it being explicit, so I'll probably update all the trait implementations to have the annotations. Especially because these can call helper functions that do need `@MainActor` and it's more obvious if the caller is annotated too.